setup.py

```diff
@@ -64,6 +64,7 @@
     "proto-plus<1.19.7",
     "pyarrow>=4,<7",
     "pydantic>=1,<2",
+    "pygments==2.12.0",
```

Comment:
What do we need this for? Should it be unpinned?

Reply:
this is for coloring the output in CLI

Reply:
More specifically it highlights the code. Since we printing out some JSON (when validation failed), it makes it more readable.